apps/web/app/pages/sbtc-rewards/components/sbtc-protocol-reward-grid.tsx

```diff
@@ -1,118 +1,155 @@
+          {pool.title}
         </styled.h3>
-        {post && (
-          <styled.div textStyle="caption.01" mt="space.01">
-            {post.prompt}
-            <LearnMoreLink destination={post.slug} precedingText={post.prompt} />
-          </styled.div>
-        )}
-        {!post && (
-          <styled.div textStyle="caption.01" mt="space.01">
-            {rewardProtocol.description}
-          </styled.div>
-        )}
+        <styled.div textStyle="caption.01" mt="space.01">
+          {pool.description}
+          <LearnMoreLink destination={legacyPostSlug[pool.id]} />
+        </styled.div>
         <Box mt="space.04">{action}</Box>
       </Box>
     </Flex>
   );
 }
 
-function TotalValueLockedCell({ rewardProtocol }: RewardProtocolCellProps): ReactElement {
-  const posts = getPosts();
-  const post = posts.totalLockedValueTvl;
+function TotalValueLockedCell({ pool }: RewardProtocolCellProps): ReactElement {
+  const {
+    concepts: { tvlConcept },
+  } = useLoaderData<typeof loader>();
+  if (!tvlConcept) return <></>;
   return (
     <ValueDisplayer
       name={
-        <PostLabelHoverCard post={post} label="Total Value Locked (TVL)" textStyle="label.03" />
+        <BasicPageHoverCard
+          title={tvlConcept?.name ?? 'Total Value Locked (TVL)'}
+          slug={tvlConcept?.relatedLegacyPost?.slug.current ?? ''}
+          description={tvlConcept?.description ?? ''}
+          label="Total Value Locked (TVL)"
+          textStyle="label.03"
+        />
       }
       value={
         <>
-          {rewardProtocol.tvl}
-          <Box textStyle="label.03">{rewardProtocol.tvlUsd}</Box>
+          {pool.tvl}
+          <Box textStyle="label.03">{pool.tvlUsd}</Box>
         </>
       }
       textAlign="left"
     />
   );
 }
 
-function HistoricalAprCell({ rewardProtocol }: RewardProtocolCellProps): ReactElement {
-  const posts = getPosts();
-  const post = posts.historicalYield;
+function HistoricalAprCell({ pool }: RewardProtocolCellProps): ReactElement {
+  const {
+    concepts: { historicalYieldConcept },
+  } = useLoaderData<typeof loader>();
+  if (!historicalYieldConcept) return <></>;
   return (
     <ValueDisplayer
-      name={<PostLabelHoverCard post={post} label="Historical yield" textStyle="label.03" />}
-      value={rewardProtocol.apr}
+      name={
+        <BasicPageHoverCard
+          title={historicalYieldConcept?.name ?? 'Historical yield'}
+          slug={historicalYieldConcept?.relatedLegacyPost?.slug.current ?? ''}
+          description={historicalYieldConcept?.description ?? ''}
+          label="Historical yield"
+          textStyle="label.03"
+        />
+      }
+      value={pool.apr}
       textAlign="left"
     />
   );
 }
 
-function MinimumCommitmentCell({ rewardProtocol }: RewardProtocolCellProps): ReactElement {
-  const posts = getPosts();
-  const post = posts.sbtcRewardsMinimumCommitment;
+function MinimumCommitmentCell({ pool }: RewardProtocolCellProps): ReactElement {
```

Comment:
```suggestion
function MinimumCommitmentCell({ pool }: RewardProtocolCellProps) {
```

no explicit typing needed